### DISCUSSION TITLE
Add setting to alter number of parallel uploads

### DISCUSSION THREAD

**Discussion by justrealmilk** [directus/directus#2767] [Feature Requests]
Add a setting to Project Settings allowing a user to alter number of parallel uploads.

This could help users on limited connections enjoy the user experience of queuing many items while also mitigating potential request timeouts.

For reference, the current maximum is 10.

**Comment by benhaynes** [comment]
@rijkvanzanten is working on Files in v9 of the Node API as we speak... interested to hear his thoughts on this.

**Reply by rijkvanzanten** [reply]
Just to confirm, are you talking about connection queueing on the front-end admin app, or in the API? @justrealmilk

**Reply by justrealmilk** [reply]
queueing on the front-end admin app ✨

**Comment by connorwinston** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
